src/selectors/entities/posts.js

```diff
@@ -391,3 +391,15 @@ export const getMostRecentPostIdInChannel = createSelector(
         return postIdsInChannel[0];
     }
 );
+
+export const getLatestReplyablePost = createSelector(
+  getPostsInCurrentChannel,
+  (posts) => {
+      for (const post of posts) {
+          if (post.state !== Posts.POST_DELETED && !isSystemMessage(post)) {
```

Comment:
This one should also check if the post is ephemeral

Reply:
Oh and you may want to return just the post id instead of the whole post, that way you can avoid a re-render if the last post has been edited

Reply:
@enahum Makes sense. I will change this to return only ID. 

BTW I copied the logic from here 
https://github.com/mattermost/mattermost-webapp/blob/f1fd8469c1d9243c4068fe56ec8d67fd9dcbac39/stores/post_store.jsx#L67.  Just wanted to check if i need to add `ephemeral` check as well?

Reply:
Well I don't really se myself replying to an ephemeral post, probably that was a bug in the webapp post store.